statistics/cmsketch.go

```diff
@@ -201,7 +194,14 @@ func (c *CMSketch) updateBytesWithDelta(bytes []byte, delta uint64) {
 }
 
 func (c *CMSketch) considerDefVal(cnt uint64) bool {
-	return cnt < 2*(c.count/uint64(c.width)) && c.defaultValue > 0
+	return (cnt == 0 || (cnt > c.defaultValue && cnt < 2*(c.count/uint64(c.width)))) && c.defaultValue > 0
+}
+
+// setValueBytes sets value of d to count.
+func (c *CMSketch) setValueBytes(d []byte, count uint64) {
+	oriCount := c.QueryBytes(d)
+	deltaCount := count - oriCount
+	c.updateBytesWithDelta(d, deltaCount)
```

Comment:
This function is inconsistent with `setValue` regarding the `defaultValues`?

Reply:
Maybe `updateValueBytes` is better?

Reply:
Why we don't update `c.defaultValue` when calling this function?